internal/syncer/connector/rest_api_test.go

```
@@ -0,0 +1,749 @@
+	if gotMethod != "GET" {
+		t.Fatalf("redirect method=%q want GET", gotMethod)
+	}
+}
+
+func TestRestAPISyncSessionPagination(t *testing.T) {
+	withRestAPITestHooks(t)
+	var requests atomic.Int32
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		requests.Add(1)
+		page := r.URL.Query().Get("page")
+		items := []any{}
```

golangci-lint (2.12.2)

[error] ineffectual assignment to items

(ineffassign)